README.md

```diff
@@ -10,8 +10,58 @@ The website uses the LAA Legal Adviser API. The code is hosted at https://github
 
 The REST API is https://prod.laalaa.dsd.io/.
 
+## Dependencies
+- [docker](https://www.docker.com/)
+
+## Installation for development
+
+### Docker
+
+Please ensure your internet network such as your VPN does not interfere with your build.
+
+To run container:
+
+`./run_local.sh`
+
+This shell script contains commands that build and runs the fala app.
+
+If you're not using `Docker Desktop`, you may want to have two terminals open.
+
+### Tips for developing with docker containers
+
+Code editing - You can edit the code on your local disk, with a local editor, as normal. (You don't have to edit the 
+files inside the Docker container, because your local directory is mounted into container.) When you save a file, it 
+becomes present in the container immediately, and the server restarts.
+
+Browsing the app - Point your local browser at http://localhost:8013/
+
+Log output - watch the output generated by the running app using: `docker logs fala -f` or `docker attach fala`
+
+To `exec` into the Fala docker container, start a new terminal and perform the following:
+
+`docker exec -it fala bash`
+
+From the shell inside the container you can run some tests e.g.
+
+`python manage.py test`
+
+Alternatively, some editors have functionality to hook into running containers, such as VS Code's 'Dev Containers' 
+extension. Docker Desktop offers a UI version of looking through logs, exec and many other interactions. 
```

Comment:
The 'Docker' extension on VS Code is much better, so might be worth updating to this

> Alternatively, some editors have functionality to hook into running containers, such as VS Code's 'Docker' 
> extension.

Reply:
The 'Docker' extension lets you attach to a shell, saving you typing `docker attach fala`. The 'Dev Containers' extension goes further, allowing you to use VS Code's debugger. Seems reasonable to mention both.